renderer.go

```diff
@@ -23,6 +24,30 @@ type formats struct {
 	clearOEL    string
 }
 
+// Returns foreground and background color code for a given color name
+func getColorFromName(colorName string) (string, string, error) {
```

Comment:
I would extend this with `, isBackground bool` just like `color.HEX` and return `(string, error)`. That way we can simplify the function below.